# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1711563990 -3600
# Parent  d91684c054060b459f0dc4100e3735bec5324ce7
Feature: Open Activity Manager in tab.

The Activity Manager is useful, but it's stand-alone window gets in the way.
Opening it in a tab by default avoids having to use the userChromeJS add-on.

diff --git a/mail/base/content/mailCore.js b/mail/base/content/mailCore.js
--- a/mail/base/content/mailCore.js
+++ b/mail/base/content/mailCore.js
@@ -618,11 +618,9 @@ function openAddonsMgr(aView) {
   });
 }
 
 function openActivityMgr() {
-  Cc["@mozilla.org/activity-manager-ui;1"]
-    .getService(Ci.nsIActivityManagerUI)
-    .show(window);
+  openContentTab("chrome://messenger/content/activity.xhtml");
 }
 
 /**
  * Open the folder properties of current folder with the quota tab selected.
